control/freqplot.py

```diff
@@ -606,11 +606,12 @@ def nyquist_plot(syslist, omega=None, plot=True, label_freq=0,
                          head_width=arrowhead_width,
                          head_length=arrowhead_length)
 
-                plt.plot(x, -y, '-', color=c, *args, **kwargs)
-                ax.arrow(
-                    x[-1], -y[-1], (x[-1]-x[-2])/2, (y[-1]-y[-2])/2,
-                    fc=c, ec=c, head_width=arrowhead_width,
-                    head_length=arrowhead_length)
+                if mirror is not False:
+                    plt.plot(x, -y, mirror, color=c, *args, **kwargs)
+                    ax.arrow(
+                        x[-1], -y[-1], (x[-1]-x[-2])/2, (y[-1]-y[-2])/2,
```

Comment:
Not related to this PR.  We should generate a new issue to check this.

Reply:
I meant the sign of the `dy` arg in the new `if mirror is not False` branch; you've change  `y[-1]` to `-y[-1]`, but haven't changed (y[-1]-y[-2])/2`.  I'd have thought if it were wrong it would be obvious on the figure, though.